tutorials/relay.md

```diff
@@ -99,66 +96,47 @@ yes
 1. Delete the management VM, you no longer need it.
```

Comment:
remove this "management VM". there is no such thing, and it was mistake to add.
It is a local unix machine (and I dont' care if the user decide to run it in a VM - locally or remote)

Reply:
Would a local Unix machine be authorized to ssh into the relay VM to change the configuration there?

Reply:
I don't care if the user uses a "management linux VM", or run it locally - but we're certainly don't SUGGEST to use such a temporary management box.
The doc can simply assume your local machine is linux (or mac) box , where all commands can easily be ran.
(the most we can do is suggest for windows user to use a VM locally)